supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc

```diff
@@ -463,6 +529,46 @@ Red Hat Directory Server conforms to LDAP standards.
 
 *See also*:
 
+[[distributed-file-system]]
+==== image:images/yes.png[yes] distributed file system (noun)
+*Description*: A file system that presents files from a number of different storage devices, potentially on many different machines and in many different locations, as a single interface to an end user or consuming service.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[[distributed-volume]]
+==== image:images/yes.png[yes] distributed volume (noun)
+*Description*: A Gluster volume that distributes data across one or more bricks.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[distributed-dispersed-volume]]
+==== image:images/yes.png[yes] distributed-dispersed volume (noun)
+*Description*: A Gluster volume that distributes and erasure codes data across one or more bricks.
```

Comment:
```suggestion
*Description*: A _distributed-dispersed volume_ is a Gluster volume that distributes and erasure codes data across one or more bricks.
```
- Also this part of the sentence seems to be missing something, or there's a typo: `that distributes and erasure codes data across one or more bricks.`